### PR TITLE
Fix the gitattributes file

### PR DESCRIPTION
**Pull request by ike709**
## About the Pull Request

@Vivalas Merry christmas

This fixes your line endings issues.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -9,6 +9,46 @@
 # force changelog merging to use union
 html/changelog.html merge=union
 
-# Declare files that will always have CRLF line endings on checkout.
-*.dm text eol=crlf
-*.dmm text eol=crlf
```
Comment (ike709) [inline .gitattributes]: The hell is wrong with you all?

Comment (Vivalas) [reply .gitattributes]: blame bay https://github.com/Baystation12/Baystation12/blob/dev/.gitattributes

**Comment by Vivalas** [comment]
I'll be freezing all PRs after this one, and implement the change once most of the ones before this are caught up.

**Comment by ivanmixo** [comment]
Why did you add literally every single label lmao

**Review by Vivalas — APPROVED** [review]
fuck it, ship it

**Pull request merged**